scenes.ts

```diff
@@ -603,6 +607,7 @@ export function initializeSceneEndpoints(state: State) {
     outgoing_url: t.string,
     place: ScenePlace,
     content: SceneContentPatch,
+    publish: t.boolean,
```

Comment:
Ah, this one should have its naming aligned as well.

Reply:
Ah yeah, missed that one. Just fixed.